activities/activity_laginhas.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+"""
+Comparing two different resample interpolation methods: nearest neighbor and bilinear.
```

Comment:
This will be little bit more complicated...I think I understand what you are trying to do, but typically you would use r.resamp.interp to resample to higher resolution, see r.resamp.interp manual page example. Moreover, now the resampling would run with whatever current region resolution will be on Tangible Landscape, but we want to avoid that. So I would suggest to modify the resolution e.g. to 1 with (put that in run_resamplediffs):

```
env2 = env.copy()
env['GRASS_REGION'] = gs.region_env(res=1, flags='a')
```
and use that env2 in run_resamplediffs. Also in main function, pass the elev_resampled into run_resamplediffs, I know you will be then resampling there and back, but this will make the examples consistent and independent on the currently set region on Tangible Landscape.

Reply:
> This is definitely a more flexible function and what I ultimately wanted to do - thank you.
> 
> I want a coarser resolution, so should I multiply instead of dividing?
> 
I don't think r.resamp.interp is meant to be used that way, you would use r.resamp.stats in that case.

> One more clarifying question, when you mention that the effect of topography is not visible here, do you mean for this particular area? If yes, I wonder if I should take the absolute values of the elev_diff and then display that instead?
> 

I think that would be a good idea.

> Thanks again!